elide-async/src/main/java/com/yahoo/elide/async/operation/GraphQLTableExportOperation.java

```diff
@@ -41,7 +41,8 @@
 
     public GraphQLTableExportOperation(TableExportFormatter formatter, AsyncExecutorService service,
             AsyncAPI export, RequestScope scope, ResultStorageEngine engine) {
-        super(formatter, service, export, scope, engine);
+        super(formatter, service, export, scope, engine,
+                        Arrays.asList(NoRelationshipsProjectionValidator.getInstance()));
```

Comment:
It seems like both need the same validators.  Should we move the NoRelationshipsProjectionValdator to the super class list?

Reply:
Single `SingleRootProjectionValidator` is must as I am taking first one from collection in call method, so kept it in superclass. `NoRelationshipsProjectionValdator` could be optional for any new subclass.